tests/std/tests/P0798R8_monadic_operations_for_std_optional/test.cpp

```diff
@@ -109,7 +110,23 @@ constexpr bool test() {
     return true;
 }
 
+void test_gh3667() {

```

Comment:
Nitpicks: We usually follow the form `test_gh_NNNN` with an extra underscore, and we usually include a comment reminding us what the bug title was.